spark/v3.0/build.gradle

```diff
@@ -80,6 +80,8 @@ project(':iceberg-spark:iceberg-spark3') {
       exclude group: 'com.google.code.findbugs', module: 'jsr305'
     }
 
+    implementation "org.roaringbitmap:RoaringBitmap"
```

Comment:
Are we adding this to all Spark versions? If so, we should cover 3.1 too. It was added recently.

Reply:
The compilation used to fail if I don't do this. But it is OK now, I will remove it.

Reply:
Looks like they still fail. My local one just don't compile them, only 3.2. I will add it to all spark 3.x versions.